tracer/src/Datadog.Trace/Configuration/IntegrationId.cs

```diff
@@ -66,6 +66,7 @@ internal enum IntegrationId
         AzureServiceBus,
         SystemRandom,
         AwsDynamoDb,
+        IbmMq,
         // If you add an integration here, please don't forget to impact the allow / block list of telemetry config
```

Comment:
Not sure which repository is `the telemetry repository`.

Reply:
Discussed offline, let me know if you need any more details 🙂